build/template-build-and-run-all-tests.yaml

```diff
@@ -81,5 +81,5 @@ jobs: #Build and stage projects
       coverageUpperThreshold: '90'
       includePartiallySucceeded: false
       fallbackOnPRTargetBranch: false
-      baseDefinitionId: '905' #this is the PR build and it is used as a baseline
+      baseDefinitionId: '1545' #this is the OB build and it is used as a baseline
```

Comment:
We cant use OneBranch build as a baseline because the value calculated is slightly different than on the PR build. Most likely due to underlying VS components.

Reply:
fixed it

Reply:
@trwalke now OB and PR build % are the same

Reply:
How was this fixed? Just curious